vars/xsDeploy.groovy

```diff
@@ -144,7 +137,7 @@ void call(Map parameters = [:]) {
 
                     dockerExecute([script: this].plus(config.docker)) {
                         xsDeployStdout = sh returnStdout: true, script: """#!/bin/bash
-                        ./piper ${parameters.verbose ? '--verbose' : ''} xsDeploy --user \${USERNAME} --password \${PASSWORD} ${operationId ? "--operationId " + operationId : "" }
+                        ./piper ${parameters.verbose ? '--verbose' : ''} xsDeploy --defaultConfig ${configFiles} --user \${USERNAME} --password \${PASSWORD} ${operationId ? "--operationId " + operationId : "" }
```

Comment:
verbose does not need to be handed over explicitly.
It is either retrieved from the config or via the parametersJSON